docs/source/installation.rst

```diff
@@ -100,6 +100,15 @@ Note that parallel HDF5 and NetCDF4 often are available as modules on
 supercomputers. Otherwise, see the respective packages for how to install
 with support for MPI.
 
+For plotting some of figures, which are generated in the Demos, `matplotlib`_ is
```

Comment:
Thanks a lot for the PR, greatly appreciated:-) 

Could you perhaps rephrase the language a little bit. Suggestion: "Some of the plots in the Demos are created using `matplotlib`_. Matplotlib is not a required dependency, but it may be easily installed from conda using"